torch/distributed/algorithms/model_averaging/averagers.py

```diff
@@ -99,15 +100,31 @@ def __init__(
             )
         self.period = period
 
-    def average_parameters(self, params):
-        r"""
-        Averages parameters if ``step`` is no less than ``warmup_steps``
+    def average_parameters(self, params: Union[Iterable[torch.nn.Parameter], Iterable[Dict[str, torch.nn.Parameter]]]):
+        """
+        Args:
```

Comment:
Nit: Put "Args" description after the overall description of this function.

Reply:
fix